src/main/java/pl/futurecollars/invoicing/handler/UserExceptionHandler.java

```diff
@@ -0,0 +1,28 @@
+package pl.futurecollars.invoicing.handler;
+
+import java.time.ZonedDateTime;
+import javax.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import springfox.documentation.spring.web.json.Json;
+
+@Slf4j
+@RequiredArgsConstructor
+@ControllerAdvice
+public class UserExceptionHandler {
+
+    @ResponseBody
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(Exception.class)
+    public ResponseEntity<Json> handleError(HttpServletRequest req, Exception ex) {
+        log.error("Request: " + req.getRequestURL() + " raised " + ex);
```

Comment:
` log.error("Request: {} raised {}",  req.getRequestURL(), ex);` is much more readable